server/interview/views.py

```diff
@@ -191,8 +195,111 @@ class NotifyInterview(APIView):
     def post(self, request):
         # TODO: Implement the notification service
         return Response({"detail": "Notification sent."})
+    
+class InterviewAll(APIView):
+    permission_classes = [IsAuthenticated]
+    # TODO: admin permission
+    def get(self, request):
+        interviews = Interview.objects.all()
```

Comment:
Why don't we just return this?

Reply:
Cause we want the technical question to display as well

Reply:
change to use serializer